src/backend/joanie/core/flows/order.py

```diff
@@ -220,7 +224,16 @@ def _post_transition_success(self, descriptor, source, target):  # pylint: disab
             target == enums.ORDER_STATE_VALIDATED
             and self.instance.product.contract_definition is None
         ):
-            self.instance.enroll_user_to_course_run()
+            try:
+                # ruff : noqa : BLE001
+                # pylint: disable=broad-exception-caught
+                self.instance.enroll_user_to_course_run()
+            except Exception as exc:
+                logger.error(
```

Comment:
This log will appear in sentry only if after in the execution an other exception is raised and not caught. If you want to be sure to open an issue in sentry then you have to capture the exception using the sentry SDK.